### PR TITLE
feat/151: add HTMLTextCounter

### PR DESCRIPTION
**Pull request by PavloIeremeiew**

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/UtilTests/HTMLTextCounterTests.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using Streetcode.BLL.Util;
+using Xunit;
+
+namespace Streetcode.XUnitTest.UtilTests
+{
+    public class HTMLTextCounterTests
```
Comment (serhiiyasenev) [inline Streetcode/Streetcode.XUnitTest/UtilTests/HTMLTextCounterTests.cs:6]: we need a test InnerText == 15k and some tags

Comment (Careless-boop) [reply Streetcode/Streetcode.XUnitTest/UtilTests/HTMLTextCounterTests.cs:6]: But for what, we are testing TextCounter not validator, why it will work with 20 characters, but not with 15k?

File: Streetcode/Streetcode.BLL/MediatR/Streetcode/Text/Create/CreateTextRequestDTOValidator.cs [hunk 2]
```diff
@@ -8,7 +9,7 @@ public CreateTextRequestDTOValidator()
     {
         RuleFor(x => x.TextCreate).NotEmpty();
         RuleFor(x => x.TextCreate.Title).NotEmpty().MaximumLength(50);
-        RuleFor(x => x.TextCreate.TextContent).NotEmpty().MaximumLength(15000);
+        RuleFor(x => x.TextCreate.TextContent).NotEmpty().Must(m => m.OnlyTextCount() <= 15000);
```
[the remark this answers is not in the document]
Comment (Careless-boop) [reply Streetcode/Streetcode.BLL/MediatR/Streetcode/Text/Create/CreateTextRequestDTOValidator.cs:12]: reasonable

File: Streetcode/Streetcode.XUnitTest/UtilTests/HTMLTextCounterTests.cs (hunk 1 above)
Comment (serhiiyasenev) [reply Streetcode/Streetcode.XUnitTest/UtilTests/HTMLTextCounterTests.cs:6]: the same reason that we test `.NotEmpty()` it's also Validator, we should check boundary values

**Review by Careless-boop — APPROVED** [review]

**Review by metrondir — APPROVED** [review]
lgtm
